hack/logextractor.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
+#
+# Usage: ./logextractor.sh [LOGS_PVC_NAME] [DEST_DIR]
+#
+
+echo "apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: logextractor
+  name: logextractor
+spec:
+  selector:
+    matchLabels:
+      run: logextractor
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: logextractor
+    spec:
+      volumes:
+      - name: logsvol
+        persistentVolumeClaim:
+          claimName: $1
+      containers:
+      - image: centos/httpd-24-centos7
+        imagePullPolicy: Always
```

Comment:
Do we need all the extras for running an http server? Can we use something slimmer like centos:7?